DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/translation/LocaleResource.java

```diff
@@ -38,6 +38,9 @@ public LocaleResource(Map<String,String> map) {
 
         if(countryKey.equals("US") && languageKey.equals("en")) {
             resource = "values" + File.separator;
+        } else if (countryKey.equals("419") && languageKey.equals("es")) {
```

Comment:
This is an unrelated change to the translation tool, but we just added Latin American Spanish to the INV Arc app, and this enables it for Android/iOS